modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -15,6 +15,53 @@ include::hello-world:partial$installation.adoc[]
+| Checksums                     | Any     | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0.sha256sum[libcouchbase-3.3.0.sha256sum]
+| Source Archive                | Any     | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0.tar.gz[libcouchbase-3.3.0.tar.gz]
+| Amazon Linux 2                | aarch64 | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_amzn2_aarch64.tar[libcouchbase-3.3.0_amzn2_aarch64.tar]
+| Amazon Linux 2                | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_amzn2_x86_64.tar[libcouchbase-3.3.0_amzn2_x86_64.tar]
+| Enterprise Linux 7            | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_centos7_x86_64.tar[libcouchbase-3.3.0_centos7_x86_64.tar]
+| Enterprise Linux 8            | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_centos8_x86_64.tar[libcouchbase-3.3.0_centos8_x86_64.tar]
+| Ubuntu 16.04 (xenial)         | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_ubuntu1604_xenial_amd64.tar[libcouchbase-3.3.0_ubuntu1604_xenial_amd64.tar]
+| Ubuntu 18.04 (bionic)         | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_ubuntu1804_bionic_amd64.tar[libcouchbase-3.3.0_ubuntu1804_bionic_amd64.tar]
+| Ubuntu 20.04 (focal)          | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_ubuntu2004_focal_amd64.tar[libcouchbase-3.3.0_ubuntu2004_focal_amd64.tar]
+| Debian 9 (stretch)            | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_debian9_stretch_amd64.tar[libcouchbase-3.3.0_debian9_stretch_amd64.tar]
+| Debian 10 (buster)            | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_debian10_buster_amd64.tar[libcouchbase-3.3.0_debian10_buster_amd64.tar]
+| Debian 11 (bullseye)          | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_debian11_bullseye_amd64.tar[libcouchbase-3.3.0_debian11_bullseye_amd64.tar]
+| Visual Studio 2015 (VC14)     | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_vc14_amd64.zip[libcouchbase-3.3.0_vc14_amd64.zip]
+| Visual Studio 2017 (VC15)     | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_vc15_amd64.zip[libcouchbase-3.3.0_vc15_amd64.zip]
+| Visual Studio 2019 (VC16)     | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_vc16_amd64.zip[libcouchbase-3.3.0_vc16_amd64.zip]
+| Visual Studio 2015 TLS (VC14) | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_vc14_amd64_openssl.zip[libcouchbase-3.3.0_vc14_amd64_openssl.zip]
+| Visual Studio 2017 TLS (VC15) | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_vc15_amd64_openssl.zip[libcouchbase-3.3.0_vc15_amd64_openssl.zip]
+| Visual Studio 2019 TLS (VC16) | x86_64  | https://packages.couchbase.com/clients/c/libcouchbase-3.3.0_vc16_amd64_openssl.zip[libcouchbase-3.3.0_vc16_amd64_openssl.zip]
+|===
+
```

Comment:
@maria-robobug, release notes on this branch don't have other releases after 3.2.0. Maybe we should remove them from here and leave link to 3.2 page?